scripts/package/macos-notarize-app.sh

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/env bash
+
+if [ -z "$1" ]; then
+    echo "Specify app bundle as first parameter"
+    exit 1
+fi
+
+if [ -z "$APPLE_ID_USER" ] || [ -z "$APPLE_ID_PASSWORD" ]; then
+    echo "You need to set your Apple ID credentials with \$APPLE_ID_USER and \$APPLE_ID_PASSWORD."
+    exit 1
+fi
+
+APP_BUNDLE=$(basename "$1")
+APP_BUNDLE_DIR=$(dirname "$1")
+
+cd "$APP_BUNDLE_DIR"
+
+# Package app for submission
+echo "Generating ZIP archive ${APP_BUNDLE}.zip..."
+ditto -c -k --rsrc --keepParent "$APP_BUNDLE" "${APP_BUNDLE}.zip"
+
+# Submit for notarization
+echo "Submitting $APP_BUNDLE for notarization..."
+RESULT=$(xcrun altool --notarize-app --type osx \
+  --file "${APP_BUNDLE}.zip" \
+  --primary-bundle-id org.musicbrainz.Picard \
+  --username $APPLE_ID_USER \
+  --password @env:APPLE_ID_PASSWORD \
+  -itc_provider MetaBrainzFoundationInc \
+  --output-format xml)
+
+if [ $? -ne 0 ]; then
+  echo "Submitting $APP_BUNDLE failed:"
+  echo $RESULT
+  exit 1
+fi
+
+REQUEST_UUID=$(echo $RESULT | xpath \
```

Comment:
Is xpath preinstalled on macs?

Reply:
I don't know if it is installed by default on any macOS or if it is part of the development tools getting installed with xcode. But I haven't installed it manually, and it is available on my install (which is a pretty bare bone macOS 10.15 with xcode installed on top) and also on the CI servers. That's all that really matters :D